src/components/Integrations/Shopify/Shopify.js

```diff
@@ -3,15 +3,24 @@ import { Helmet } from 'react-helmet';
 import Loading from '../../Loading/Loading';
 import { InjectAppServices } from '../../../services/pure-di';
 import logo from './logo.svg';
-import './Shopify.css';
 import { FormattedHTMLMessage, FormattedMessage, injectIntl } from 'react-intl';
+import styled from 'styled-components';
 
 const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
   const [shops, setShops] = useState([]);
   const [isConnected, setIsConnected] = useState(null);
   const [error, setError] = useState(null);
   const [isLoading, setIsLoading] = useState(true);
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const ShopifyLogo = ({ className, svglogo }) => (
```

Comment:
Since this component is named _ShopifyLogo_, what do you think about hardcoding the src in place of accepting it by parameter?

```suggestion
  const ShopifyLogo = ({ className }) => (
```

BTW, the logo is already hardcoded 

```
<img className={className} src={logo} alt="Shopify logo" />
                                ^^^^ It saids logo (hardcoded) in place of svglogo (parameter) , I personally prefeer hardcoded one.
```